presto-hive/src/main/java/io/prestosql/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -399,7 +448,18 @@ private void invokeNoMoreSplitsIfNecessary()
     {
         ListenableFuture<?> lastResult = COMPLETED_FUTURE;
         for (InputSplit inputSplit : targetSplits) {
+            String payload = null;
+            if (tracer.isEnabled()) {
+                Map<String, Object> map = new HashMap<>();
+                map.put("path", ((FileSplit) inputSplit).getPath().toString());
+                map.put("start", ((FileSplit) inputSplit).getStart());
+                map.put("length", ((FileSplit) inputSplit).getLength());
+                payload = jsonCodec.toJson(map);
+                tracer.emitEvent(ConnectorTracerActionType.LOAD_SPLIT_START, payload);
+            }
             Optional<InternalHiveSplit> internalHiveSplit = splitFactory.createInternalHiveSplit((FileSplit) inputSplit);
+            tracer.emitEvent(ConnectorTracerActionType.LOAD_SPLIT_END, payload);
```

Comment:
`isEnabled` for uniformity, it won't matter in terms of correctness. Thinking if we could do something better instead of double checking for  `isEnabled`.